pallets/liquidity-pools/src/mock.rs

```diff
@@ -27,6 +27,15 @@ pub const ALICE_32: [u8; 32] = [2; 32];
 pub const ALICE: AccountId = AccountId::new(ALICE_32);
 pub const ALICE_ETH: [u8; 20] = [2; 20];
 pub const ALICE_EVM_DOMAIN_ADDRESS: DomainAddress = DomainAddress::EVM(42, ALICE_ETH);
+pub const ALICE_EVM_LOCAL_ACCOUNT: AccountId = {
```

Comment:
Nit - maybe you can use something like `as_h160_32bytes` which we use for integration tests.

Reply:
NOTE: Luckly this could be remove once `DomainAddress` knows directly about conversions

Reply:
> Nit - maybe you can use something like `as_h160_32bytes` which we use for integration tests.

Not sure if we want to add runtime integration tests as dev dependency to `pallet-liquidity-pools`. Given @lemunozm upcoming `DomainAddress` refactor, I would leave as is with a comment to refactor as part of the PR or afterwards. Does that sound reasonable to you @cdamian ?

Reply:
Sure.